modules/services/screen-locker.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.screen-locker;
+
+in {
+
+  options.services.screen-locker = {
+    enable = mkOption {
```

Comment:
I'd prefer `enable = mkEnableOption "screen locker for X session";`.

Reply:
Copy-pasted from `modules/services/redshift.nix`, sorry :)